TelemetrySite/server/Context/context.py

```diff
@@ -0,0 +1,135 @@
+                if not "selected" in currentConfigData:           
+                    idBikeConfig.append(utils.exec_commit_with_id(sqlCommands[index], self.DictToTuple(currentConfigData))[0][0])
+                else:
+                    #execute a query to find the id where the name is the same as the passed name
+                    query = f"SELECT id FROM {self.configNames[index]} WHERE configName = '{currentConfigData["selected"]}'"
+                    
+                    if currentConfigData["selected"]=="":
+                        if index == 1:
+                            hasIMU=False
+                        else:
+                            hasTMU=False
+                        continue
+                    idBikeConfig.append(utils.exec_get_one(query, [0,])[0])
+        #establish sql commands for all non event, bike, and context configs
+        #save the event and bike ids for to reference in the context db
+
+        sqlEvent = "INSERT into Event (id, eventDate, eventType, location) VALUES (DEFAULT, %s, %s, %s) RETURNING id"
+
+        eventAndBikeId.append(utils.exec_commit_with_id(sqlEvent, self.DictToTuple(contextValue["Event"]))[0][0])
+
+        #if the bike isn't a saved value
+        if not bikeSaved:
+            #create a new one
+            sqlBikeConfig = "INSERT into BikeConfig (id,  platformName, tirePressure, coolantVolume, configName, bmsConfigId, imuConfigId, tmuConfigId, tmsConfigId, pvcConfigId, mcConfigId) VALUES (DEFAULT,  %s, %s, %s, %s, %s," 
+            #set imu and tmu to default as needed
+            if hasIMU is True:
+                sqlBikeConfig+="%s, "
+            else:
+                sqlBikeConfig+="DEFAULT, "
+
+            if hasTMU is True:
+                sqlBikeConfig+="%s, "
+            else:
+                sqlBikeConfig+="DEFAULT, "
+            
+            sqlBikeConfig+="%s, %s, %s) RETURNING id"
+            eventAndBikeId.append(utils.exec_commit_with_id(sqlBikeConfig, self.DictToTuple(contextValue["BikeConfig"])+tuple(idBikeConfig))[0][0])
+
+        else:
+            # if it is saved, get the id
+            sqlCommand= f"SELECT id from BikeConfig WHERE configName = '{contextValue["BikeConfig"]["selected"]}'"
+            eventAndBikeId.append(utils.exec_get_one(sqlCommand, [0,])[0])
+
+        sqlContext = "INSERT into Context (id, airTemp, humidity, windSpeed, windAngle, riderName, riderWeight, driverFeedback, distanceCovered, startTime, eventId, bikeConfigId) VALUES (DEFAULT, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s) RETURNING id"        
+
+        utils.exec_commit_with_id(sqlContext, self.DictToTuple(contextValue["MainBody"])+tuple(eventAndBikeId))
+
+        # Respond back to the client. 201 code for created
+        return jsonify({"message": "Data received successfully", "received": data}), 201
+    
+
+    def post(self):
+
+
+        return jsonify(["Post Called"])
+    
+    def delete(self):
+
+        
```

Comment:
idk if these are purposefully like, left blank to do later/missing implementations. if so at least add a comment saying these are supposed to be doing something in the future. Otherwise dont have these double empty lines between the function name and the body.